example/index.ts

```diff
@@ -31,7 +31,8 @@ class TestStack extends Stack {
 
     const watchful = new Watchful(this, 'watchful', {
       alarmEmail: 'benisrae@amazon.com',
-      alarmSqs: 'arn:aws:sqs:us-east-1:444455556666:alarm-queue'
+      alarmSqs: 'arn:aws:sqs:us-east-1:444455556666:alarm-queue',
+      alarmSns: 'arn:aws:sns:us-east-2:444455556666:MyTopic'
```

Comment:
TBH I don't understand why these are not `IQueue` and `ITopic` instead of string ARNs... Shall we change (and major version bump)?

Reply:
Yeah I was thinking about that too, the only reason I did a string was to stay consistent with what was already there. i.e. passing in strings. I'll update

Reply:
Updated! 😄